app/models/question.rb

```diff
@@ -9,11 +9,22 @@ def create_tags(tags)
       self.tags.create(title: tag.strip)
     end
   end
-  
+
   validates :title, presence: true
-  
+
   def self.search(query)
-    where("title like ?", "%#{query}%") 
+    where("title like ?", "%#{query}%")
+  end
+
+  def get_question_tags
```

Comment:
I feel like this could be accomplishable with only Active Record query mechanism....